docker-compose.yaml

```diff
@@ -0,0 +1,153 @@
+version: '3.7'
+services:
+
+  #
+  # CNaaS NMS interactive frontend development setup
+  #
+
+
+  cnaas_front:
```

Comment:
I have a nitpick about the naming of services in docker compose (My nitpick is the same for cnaas-nms itself, but this is our contribution, so I'll add it here):

I don't understand the need to prefix all the services with `cnaas_`. Docker compose uses the current directory name as its context (unless you specifically override it in `docker-compose.yml`). The context name will be part of the container names anyway, and I think it is quite obvious what project directory you're in when you are interating with Docker Compose.

Using `cnaas_front` as the service name will cause Docker Compose to create a container named `cnaas-front_cnaas_front_1`, which contains a lot of redundancies. Just naming the service `front` would lead to the container being namd `cnaas-front_front_1`, which is easier to read as "The `front` service of the `cnaas-front` context".

Reply:
Agreed on the naming thing. Good catch! Not sure I'll have time to fix that though (as the README will need fixing as well).